source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -701,8 +701,13 @@ void Graph::selectMaterial(UiNodePtr uiNode)
 // set the node to display in render veiw based off the selected node or nodegraph
 void Graph::setRenderMaterial(UiNodePtr node)
 {
+    // For now surface shaders and materials are considered renderable.
+    // This can be adjusted as desired to includ being able to use outputs,
+    // and / a sub-graph in the nodegraph.
+    mx::StringSet renderableTypes = { "material", "surfaceshader" };
```

Comment:
For clarity, I'd recommend making this a constant StringSet named RENDERABLE_TYPES.